### DISCUSSION TITLE
Question: Can BullMQ be used with Node-Red?

### DISCUSSION THREAD

**Discussion by wernermorgenstern** [taskforcesh/bullmq#2131] [Q&A]
**Is your feature request related to a problem? Please describe.**
We are redesigning and rewriting our Node-Red Services, to be able to scale horizontally and get more performance out of it.
So we are going to use BullMQ to send messages between the services, and use the queues to add Jobs, and then workers.
We have an webapi service (with Node-Red), which handles API Calls.  To rewrite that one will take a bit more time.
So for the time being, we still want to use BullMQ, but have the worker and adding jobs to the BullMQ queue within the Node-Red.

So I was wondering if that is possible.

**Comment by manast** [comment]
Sorry for the late response. I am not familiar with Node-Red so I do not know how feasible it is to integrate BullMQ with it, but after watching some videos I guess it should be possible. If you encounter any hindrances please share them with us.
